tests/test_sysfs.sh

```diff
@@ -0,0 +1,99 @@
+# Error on unset variables
+set -u
+
+# Load Linux version of _lp_battery()
+uname() { printf 'Linux'; }
+
+. ../liquidprompt --no-activate
+unset -f uname
+
+LP_ENABLE_BATT=1
+_LP_BATTERY_FUNCTION=__lp_battery_sysfs
+
+typeset -a battery_types battery_presents battery_status battery_capacities battery_out_statuses battery_values
+
+# Add test cases to these arrays like below
+
+# Linux 5.11.0-38-generic #42~20.04.1-Ubuntu SMP Tue Sep 28 20:41:07 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
+battery_types+=("Battery")
+battery_presents+=("1")
+battery_statuses+=("Discharging")
+battery_capacities+=("67")
+battery_out_statuses+=(0)
+battery_values+=("67")
+battery_scopes+=("")
+
+# Full, not charging
+battery_types+=("Battery")
+battery_presents+=("")
+battery_statuses+=("")
+battery_capacities+=("")
+battery_out_statuses+=(4)
+battery_values+=("")
+battery_scopes+=("")
+
+# Not a battery
+battery_types+=("Mains")
+battery_presents+=("")
+battery_statuses+=("")
+battery_capacities+=("")
+battery_out_statuses+=(4)
+battery_values+=("")
+battery_scopes+=("")
+
+# Wrong type of battery battery (Wireless mouse, some other)
+battery_types+=("Battery")
+battery_presents+=("")
+battery_statuses+=("")
+battery_capacities+=("")
+battery_out_statuses+=(4)
+battery_values+=("")
+battery_scopes+=("Device")
```

Comment:
These cases look bare to me. I would imagine that most of these should have real values in them. I would have at least one with a lot of empty values, to test that Liquidprompt correctly ignores it, but you should also show actual real world valid cases as well.

Reply:
Actually, when I left the value blank, that means the file doesn't exist. I will double check on my machine, and maybe one other, and look through the documentation, but I'm pretty confident that this is a consistent behaviour.

I will also do the fixes you requested, and touch up the documentation, probably sometime tomorrow evening, or something like that.